src/UsdPlusMinter.sol

```diff
@@ -139,6 +139,7 @@ contract UsdPlusMinter is IUsdPlusMinter, UUPSUpgradeable, AccessControlDefaultA
     {
         if (receiver == address(0)) revert ZeroAddress();
         if (paymentTokenAmount == 0) revert ZeroAmount();
+        // Additional check to prevent deposit from happening to itself
```

Comment:
Is it a security concern if users set the minter contract as the receiver, given tokens can be transferred out and no functions can be exploited by having tokens in the contract?